src/Microsoft.Sbom.Extensions/ISbomParser.cs

```diff
@@ -42,5 +43,5 @@ public interface ISbomParser
     /// Returns the compliance standard set by the user, if it is valid.
     /// </summary>
     /// <param name="spdxVersion"></param>
-    public void SetComplianceStandard(string? complianceStandardFromCli);
+    void SetComplianceStandard(ComplianceStandardType complianceStandard);
```

Comment:
`public` keyword is required for latest standards
```suggestion
    publci void SetComplianceStandard(ComplianceStandardType complianceStandard);
```